lib/fluent/plugin/in_prometheus.rb

```diff
@@ -23,13 +40,39 @@ def configure(conf)
 
     def start
       super
-      @server = WEBrick::HTTPServer.new(
+      config = {
         BindAddress: @bind,
         Port: @port,
         MaxClients: 5,
         Logger: WEBrick::Log.new(STDERR, WEBrick::Log::FATAL),
         AccessLog: [],
-      )
+      }
+      unless @ssl.nil? || !@ssl['enable']
```

Comment:
use `if` instead. `if @ssl && @ssl['enable']` ?

Reply:
I was unable to achieve what I expected using the 'if' form.  There always seemed to be a need to test against `nil?`

Reply:
Ok!